<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>620</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Jussie's FTP Server</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="iconSize">
   <size>
    <width>45</width>
    <height>45</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="logTab">
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="log">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="usersTab">
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Connected Users</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableWidget" name="connectedUsersTable">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Username</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Connected</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_connection"/>
   <addaction name="actionClose_connection"/>
   <addaction name="actionConfigure_server"/>
   <addaction name="separator"/>
   <addaction name="actionUser_list"/>
   <addaction name="actionAdd_user"/>
   <addaction name="separator"/>
   <addaction name="actionSave_log"/>
   <addaction name="actionClear_current_log"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionOpen_connection"/>
    <addaction name="actionClose_connection"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure_server"/>
   </widget>
   <widget class="QMenu" name="menuUsers">
    <property name="title">
     <string>Users</string>
    </property>
    <addaction name="actionUser_list"/>
    <addaction name="actionAdd_user"/>
   </widget>
   <widget class="QMenu" name="menuLogging">
    <property name="title">
     <string>Logging</string>
    </property>
    <addaction name="actionSave_log"/>
    <addaction name="actionClear_current_log"/>
   </widget>
   <addaction name="menuServer"/>
   <addaction name="menuUsers"/>
   <addaction name="menuLogging"/>
  </widget>
  <action name="actionOpen_connection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/openCon.png</normaloff>:/icons/icons/openCon.png</iconset>
   </property>
   <property name="text">
    <string>Open connection</string>
   </property>
  </action>
  <action name="actionClose_connection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/closeCon.png</normaloff>:/icons/icons/closeCon.png</iconset>
   </property>
   <property name="text">
    <string>Close connection</string>
   </property>
  </action>
  <action name="actionUser_list">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/users-group.png</normaloff>:/icons/icons/users-group.png</iconset>
   </property>
   <property name="text">
    <string>User list</string>
   </property>
  </action>
  <action name="actionAdd_user">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/new-user.png</normaloff>:/icons/icons/new-user.png</iconset>
   </property>
   <property name="text">
    <string>Add user</string>
   </property>
  </action>
  <action name="actionSave_log">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/log.png</normaloff>:/icons/icons/log.png</iconset>
   </property>
   <property name="text">
    <string>Save log</string>
   </property>
  </action>
  <action name="actionClear_current_log">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/clearlog.png</normaloff>:/icons/icons/clearlog.png</iconset>
   </property>
   <property name="text">
    <string>Clear current log</string>
   </property>
  </action>
  <action name="actionConfigure_server">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/configure.png</normaloff>:/icons/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configure server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
